src/indices.jl

```diff
@@ -34,3 +34,17 @@ end
 
 istrivialpermutation(p::IndexTuple) = p == trivialpermutation(p)
 istrivialpermutation(p::Index2Tuple) = p == trivialpermutation(p)
+
+"""
+    const LabelType = Union{Int,Symbol,Char}
+
+Alias for supported label types.
+"""
+const LabelType = Union{Int,Symbol,Char}
+
+"""
+    const Labels{I<:LabelType} = Union{Tuple{Vararg{I}},Vector{I}}
+
+Alias for supported label containers.
+"""
+const Labels{I<:LabelType} = Union{Tuple{Vararg{I}},AbstractVector{I}}
```

Comment:
This excludes things like `[1,3,:x,:a]` but I guess that is acceptable. We can always recommend tuples, as `(1,3,:x,'a') isa Labels` is still true.

Reply:
`collect(LabelType, [1, 2, :x, :a])` should also still work in that case :)